### PR TITLE
Chaos crate

### PR DESCRIPTION
**Pull request by Bawhoppen**
# PR MADE ON BEHALF OF ma44

Chaos crate, hidden, gives a random 1 of 6 types of clusterbangs. 2500 cargo points.

### CODE REVIEW

**Comment by ma44** [comment]
pls don't be mean to this guy because of me, were here for constructive criticism

**Comment by Niknakflak** [comment]
remove var/ from line 1386

**Comment by GunHog** [comment]
What if there were...**3** of them in the crate?

**Comment by Dorsisdwarf** [comment]
Uh, this seems like cargo are going to fuck things up a _lot_ more. Hidden as in emag or contraband?

**Comment by Bawhoppen** [comment]
hidden means BROAD on the circuitboard iirc

**Comment by Bawhoppen** [comment]
niknak is a fucking idiot holy cuck i cannot believe i fell for his lies

**Comment by Dorsisdwarf** [comment]
I'd have put this behind the emag and removed the smoke, myself. Cluster-acid is brutal shit for crew gear.

**Comment by Niknakflak** [comment]
Actually, if you read what I said, I said remove the /var from line 1386

**Comment by GeneralGoose** [comment]
it has 6 grenades, not 1

**Comment by AdamElTablawy** [comment]
Inferno, emp, and acid are instant shuttle calls if used, esp considering facid is the old facid cloud and scores instakills and the inferno is hell on adminlogs

Should probably cost way more than 2500 points for an rng fuck the world crate

Like iunno null crate tier

File: code/modules/cargo/packs.dm [hunk 1]
```diff
@@ -70,6 +70,21 @@
 	crate_name = "internals crate"
 	crate_type = /obj/structure/closet/crate/internals
 
+/datum/supply_pack/emergency/Chaos
```
Comment (ChangelingRain) [inline code/modules/cargo/packs.dm]: I forget but I don't think that should be capitalized it might break something

**Comment by anconfuzedrock** [comment]
singulo is an instant shuttle call too

File: code/modules/cargo/packs.dm [hunk 2]
```diff
@@ -70,6 +70,21 @@
...
+	name = "Chaos Crate"
+	hidden = TRUE
+	cost = 2500
+	var/num_contained = 1
+	contains = list(/obj/item/weapon/grenade/clusterbuster/emp,
+	/obj/item/weapon/grenade/clusterbuster/smoke,
+	/obj/item/weapon/grenade/clusterbuster/metalfoam,
+	/obj/item/weapon/grenade/clusterbuster/inferno,
+	/obj/item/weapon/grenade/clusterbuster/teargas,
+	/obj/item/weapon/grenade/clusterbuster/facid)
```
Comment (KorPhaeron) [inline code/modules/cargo/packs.dm]: lol

**Comment by KorPhaeron** [comment]
lets not get carried away

**Comment by Bawhoppen** [comment]
I am sorry @Niknakflak I don't know why I was being so mean to you

**Comment by PKPenguin321** [comment]
You said the forbidden word in a commit message. The github timecops are on the way. Watch your back

**Pull request closed without merging**